boards/esp32-wrover-kit/Makefile.include

```diff
@@ -1,6 +1,7 @@
 PSEUDOMODULES += esp32_wrover_kit_camera
 
 USEMODULE += boards_common_esp32
+USEMODULE += esp_rtc_timer_32k
```

Comment:
Do we always want to build the RTC?
I'd rather put 
```Make
ifneq (,$(filter periph_rtc,$(USEMODULE)))
  USEMODULE += esp_rtc_timer_32k
endif
```
in the board's `Makefile.dep`. Or maybe add a new `FEATURES_PROVIDED`.

Reply:
`esp_rtc_timer_32` is just a pseudomodule to control the compilation in `cpu/esp32/periph/rtc.c` and doesn't depend on `periph_rtc`. RTC is only compiled if module `periph_rtc` is used.

Reply:
Ah I see - but then it would probably be better semantics to set
```
FEATURES_PROVIDED += esp_rtc_timer_32k
```

Reply:
However, if I am right, we would need an additional `FEATURES_OPTIONAL` in MCU's `Makefile.include` to use the module when the freature is provided.

Reply:
Yes and then check for `FEATURES_USED` to select the the proper implementation.
Maybe a bit more convoluted, but certainly the 'cleaner' solution.